templates/default/rdrs-start.sh.erb

```diff
@@ -1,5 +1,20 @@
 #!/usr/bin/env sh 
 
+
+<% if node['ndb']['rdrs']['containerize'].eql? "true" -%>
+
+rm -f <%= node['ndb']['log_dir'] %>/rdrs.cid 
+docker run --rm  \
+  --name rdrs-<%= node['ndb']['version'] %>.service  \
```

Comment:
I believe the version in the container name is redundant

Reply:
there is no "belief" in engineering

Reply:
@smkniazi Let me put it in another way then :) I have never seen a systemd service or docker container where the name contains the version

Reply:
Yeah because we need to handle update and when we update we shutdown the previous version.

Reply:
it is removed